.github/workflows/ci.yml

```diff
@@ -134,4 +134,18 @@ jobs:
           cd build
           python3 ../configure.py --enable-optimize --metamod=${{ env.DEPENDENCIES_ROOT_WIN }}\metamod-am --hlsdk=${{ env.DEPENDENCIES_ROOT_WIN }}\hlsdk --mysql=${{ env.DEPENDENCIES_ROOT_WIN }}\mysql-5.5
           ambuild
-          
+      - name: Tar files
```

Comment:
All of these CI changes need to be a separate PR. That said, we must only publish artifacts that are created (1) by buildbot and (2) contain code reviewed by a team member. Any artifacts that do not follow these preconditions must be clearly marked as third-party/untrusted builds.

Reply:
removed